api/app/Exceptions/Handler.php

```diff
@@ -38,9 +38,41 @@ public function report(Exception $e)
     public function render($request, Exception $e)
     {
 
-        return BaseController::renderException($request, $e, env('APP_DEBUG', false));
+        $debug = env('APP_DEBUG', false);
+
+        $message = $e->getMessage();
+        if (!$message){
+            $message = 'An error occurred';
+        }
+
+        $debugData = [
+            'exception' => get_class($e),
+            'message' => $e->getMessage(),
+            'code' => $e->getCode(),
+            'file' => $e->getFile(),
+            'line' => $e->getLine(),
+            'trace' => explode("\n", $e->getTraceAsString()),
+        ];
+
+        $response = [
+            'message' => $message,
+        ];
+
+        $statusCode = 500;
+
+        if ($e instanceof HttpExceptionInterface){
+            $statusCode = $e->getStatusCode();
+
+            if (method_exists($e, 'getResponse')) {
+                $response = $e->getResponse();
+            }
+        }
+
+        if ($debug){
+            $response['debug'] = $debugData;
+        }
 
-//        return parent::render($request, $e);
+        return response()->json($response, $statusCode, array(), JSON_PRETTY_PRINT);
```

Comment:
Yea good move bringing this back - we were originally relying on some methods in the `BaseController`